src/Sparrow/ByteString.cs

```diff
@@ -626,7 +626,7 @@ public class SegmentStack : StackHeader<UnmanagedGlobalSegment>
 
     public sealed class ByteStringContext : ByteStringContext<ByteStringMemoryCache>
     {
-        public const int MinBlockSizeInBytes = 64 * 1024; // If this is changed, we need to change also LogMinBlockSize.
+        public const int MinBlockSizeInBytes = 4 * 1024; // If this is changed, we need to change also LogMinBlockSize.
```

Comment:
@redknightlois Can you suggest what change is required in `LogMinBlockSize`?

Reply:
The calculation is Log(2, 4*1024) ... those are the actual potential buckets you could have given the minimum block size in bytes we are defining.

Reply:
Okay, so we should change it to 12